packages/openapi-ts/src/generate/client.ts

```diff
@@ -3,13 +3,15 @@ import { createRequire } from 'node:module';
 import path from 'node:path';
 
 import { getConfig, isStandaloneClient } from '../utils/config';
+import { appendExt } from '../utils/extension';
 import { ensureDirSync } from './utils';
 
 const require = createRequire(import.meta.url);
 
 export const clientModulePath = () => {
   const config = getConfig();
-  return config.client.bundle ? './client' : config.client.name;
+
+  return appendExt(config.client.bundle ? './client' : config.client.name);
```

Comment:
Do we need to add extension to node_modules imports too? This may generate code like `'@hey-api/client-fetch.ts'`

Reply:
No, only for relative/absolute paths or importing specific files.

I see, Id have to check then how to differentiate if its coming from core or an external package...